docs/setup.asciidoc

```diff
@@ -271,16 +280,19 @@ as shown in the following example:
     </system.webServer>
 </configuration>
 ----
++
+NOTE: To learn more about adding modules, see the https://docs.microsoft.com/en-us/iis/configuration/system.webserver/modules/add[Microsoft docs].
 
-You can also configure the agent using `web.config` as described at <<configuration-on-asp-net>>.
-
-The `ElasticApmModule` instantiates the APM agent on first initialization. There are some scenarios however where
-you might want to control agent instantiatiation, such as configuring filters in application start. The
-`ElasticApmModule` exposes a `CreateAgentComponents()` method that returns agent components configured to work with
-ASP.NET Full Framework, that can then be used to instantiate the agent.
-
-For example, one might wish to add transaction filters to the agent in application start
-
+. Recompile your application and deploy it.
++
+The `ElasticApmModule` instantiates the APM agent on the first initialization. However, there may be some scenarios where
+you want to control the agent instantiatiation, such as configuring filters in the application start.
```

Comment:
```suggestion
you want to control the agent instantiation, such as configuring filters in the application start.
```

Reply:
Thanks!

Reply:
I had to look at this word for entirely too long before I could tell that it was misspelled. I feel like it could be in the chorus of a song:

> 🎵 There may be some scen-aaaaaa-rios 🎵
> 🎵 where you want to control the agent -- the agent -- the agent 🎵
> 🎵 instantiatiation, instantiatiation, instantiatiation 🎵

Reply:
Love it!